src/electionguard_gui/web/components/admin/view-election-component.js

```diff
@@ -28,54 +34,69 @@ export default {
   template: /*html*/ `
     <div v-if="election">
       <div class="container">
-        <div class="row">
-          <div class="col col-10">
+        <div class="row mb-4">
+          <div class="col">
             <h1>{{election.election_name}}</h1>
           </div>
-          <div class="col col-2 text-end">
-            <a :href="getEncryptionPackageUrl()" class="btn btn-primary">
+          <div class="col col-xs-3 text-end">
```

Comment:
I don't love this UI for downloading the encryption package and uploading ballots.  Open to suggestions.